File: pbwebcert/pbwebcert-jar/src/main/java/it/csi/pbandi/pbwebcert/integration/request/LanciaPropostaRequest.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbwebcert.integration.request;


import java.util.List;

import it.csi.pbandi.pbwebcert.dto.certificazione.CodiceDescrizione;
import it.csi.pbandi.pbwebcert.dto.certificazione.PropostaCertificazioneDTO;

public class LanciaPropostaRequest {
	private PropostaCertificazioneDTO proposta;
	private List<CodiceDescrizione> lineeDiInterventoDisponibili;

	public PropostaCertificazioneDTO getProposta() {
		return proposta;
	}

	public void setProposta(PropostaCertificazioneDTO proposta) {
		this.proposta = proposta;
	}

	public List<CodiceDescrizione> getLineeDiInterventoDisponibili() {
		return lineeDiInterventoDisponibili;
	}

	public void setLineeDiInterventoDisponibili(List<CodiceDescrizione> lineeDiInterventoDisponibili) {
		this.lineeDiInterventoDisponibili = lineeDiInterventoDisponibili;
	}
}

File: pbwebcertwcl/src/app/crea-proposta/commons/requests/lancia-proposta-request.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

import { CodiceDescrizione } from "src/app/shared/commons/dto/codice-descrizione";
import { PropostaCertificazioneDTO } from "src/app/shared/commons/dto/proposta-certificazione-dto";

export class LanciaPropostaRequest {
    constructor(
        public proposta: PropostaCertificazioneDTO,
        public lineeDiInterventoDisponibili: Array<CodiceDescrizione>
    ) { }
}